addons/im_livechat/static/src/core/common/thread_actions.js

```diff
@@ -0,0 +1,17 @@
+import { patch } from "@web/core/utils/patch";
+
+import { threadActionsInternal } from "@mail/core/common/thread_actions";
+
+patch(threadActionsInternal, {
+    condition(component, id, action) {
+        const visitorActions = ["fold-chat-window", "close", "restart", "settings"];
+        if (
+            component.thread?.channel_type === "livechat" &&
+            !component.store.self.isInternalUser &&
+            !visitorActions.includes(id)
+        ) {
+            return false;
+        }
+        return super.condition(component, id, action);
+    },
```

Comment:
We can remove `allowedThreadActions` and code below of `disabled_features.js`, since this does the same. This code is not exported so their removal follows stable policy.

Disabling thread actions in `disabled_features.js` was the old way to do before we had `threadActionsInternal` (or equivalent in 19.0 with `patch(ThreadAction, { _condition() {} })`)

The implementation in `disabled_features.js` was fine as long as this is a persistent technique to whitelist thread actions in the given bundle, which works with `embed/` but it doesn't in public bundle because condition depends on `isInternalUser`.